report/chapters/problem_analysis.typ

```diff
@@ -7,6 +7,29 @@
 == PACT
 
 == FACTOR
+We have applied the FACTOR model to the absence calendar system to ensure a comprehensive analysis of the key components and requirements needed for successful implementation. The FACTOR model provides a structured approach to identifying the system’s functionality, the domain in which it operates, the conditions of development, the technologies involved, the objects that are central to the system, and the overall responsibilities. By using this model, we aim to clearly define how our system will function within the organizational context of KMD.

```

Comment:
Thought we agreed not to use we in the report, unsure though. Good section, maybe add that it also serves as a way to communicate exactly between KMD and us